2023-09-15 16:04:16.738 [Error] {"ExceptionMessage":"An error occurred while saving the entity changes. See the inner exception for details.","MethodName":"AddAsync","User":"?","Parameters":[{"Name":"entity","Value":{"Content":"naber","TeamId":1,"UserId":1},"Type":"CommentAddDto"}]}
2023-09-15 17:35:11.270 [Error] {"ExceptionMessage":"An error occurred while saving the entity changes. See the inner exception for details.","MethodName":"AddAsync","User":"?","Parameters":[{"Name":"entity","Value":{"Name":"admin","CreatedBy":0,"CreatedDate":"2023-09-15T14:34:54.212Z","UpdatedBy":0,"UpdatedDate":"2023-09-15T14:34:54.212Z","Id":1},"Type":"OperationClaim"}]}
2023-09-15 17:35:36.760 [Error] {"ExceptionMessage":"An error occurred while saving the entity changes. See the inner exception for details.","MethodName":"AddAsync","User":"?","Parameters":[{"Name":"entity","Value":{"Name":"admin","CreatedBy":1,"CreatedDate":"2023-09-15T14:34:54.212Z","UpdatedBy":1,"UpdatedDate":"2023-09-15T14:34:54.212Z","Id":1},"Type":"OperationClaim"}]}
